tests/scancode/test_ignore_files.py

```diff
@@ -74,3 +81,45 @@ def test_ignore_glob_file(self):
             {}
         )
         assert is_ignored(*test)
+
+    def test_resource_paths_1(self):
+        test = (
+            test_dir,
+            {'*.doc': 'test ignore'}
+        )
+        expected = [
+            'user',
+            'user/src',
+            'user/src/test',
+            'user/src/test/sample.txt'
+        ]
+        for abs_path, rel_path in resource_paths(*test):
```

Comment:
Also why testing in a loop? Make this a list and assert that the list is what you expect as a whole.